src/develop/imageop.c

```diff
@@ -2267,14 +2267,24 @@ void dt_iop_clip_and_zoom_demosaic_half_size(float *out, const uint16_t *const i
 
           if(!((pc >= 60000) ^ (MAX(MAX(p1, p2), MAX(p3, p4)) >= 60000)))
           {
-            sum = _mm_add_epi32(sum, _mm_set_epi32(0, p4, p3 + p2, p1));
+            if(filters == 0xb4b4b4b4 || filters == 0x9c9c9c9c) // CYGM or RGBE so lets keep all four values
+              sum = _mm_add_epi32(sum, _mm_set_epi32(p3, p4, p2, p1));
```

Comment:
The order seems to be the correct one for CYGM. But thanks for pointing it out as that could be why there's a difference in the RGBE case between full-pipe and MIPF. This branch may need to be three way, splitting the 0xb4 and 0x9c cases.